dist/images/ovnkube.sh

```diff
@@ -1030,14 +1059,15 @@ ovn-node() {
     ${ovn_unprivileged_flag} \
     --nodeport \
     --mtu=${mtu} \
-    ${OVN_ENCAP_IP} \
+    ${ovn_encap_ip_flag} \
```

Comment:
why this change?

Reply:
This exposes a way to explicitly set OVN encapsulation IP.

A first time run of ovnkube node will derrive this IP address from `util.GetNodePrimaryIP()` which takes the IP address of the k8s node object.

In case ovnkube node runs on Smart-NIC,  the OVN encap IP != Node IP

Reply:
thank make sense